assets/files.wxs

```diff
@@ -1646,6 +1646,9 @@
         <Component Id="cmp770700CA06564FB187D1CD2A79AA3DA8" Guid="{c244e7e1-3cb0-4880-aba5-e2be4fd78fbf}">
```

Comment:
@bergmeister, your last commit had 2 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/learning-powershell/powershell-beginners-guide.md.https://channel9.msdn.com/Series/GetStartedPowerShell3/06 should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 112
112:                                 throw "retry of URL failed with error: $($_.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/learning-powershell/powershell-beginners-guide.md.https://channel9.msdn.com/events/MMS/2012/SV-B406 should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 112
112:                                 throw "retry of URL failed with error: $($_.Message)"

```